webapp/packages/plugin-data-spreadsheet-new/src/styles/base.scss

```diff
@@ -6,46 +6,48 @@
  * you may not use this file except in compliance with the License.
  */
 
-.cb-react-grid-container {
-  outline: 0;
-  display: flex;
-  flex-direction: column;
-  flex: 1;
-  overflow: auto;
-  user-select: none;
-  -webkit-user-select: none;
+:global {
+  .cb-react-grid-container {
+    outline: 0;
```

Comment:
this file is not a module css file, it shouldn't have to been treated as css module (webpack configuration?)